test/benchmarks/ecpairing.inputs

```diff
@@ -1,5 +1,23 @@
+empty
+ecpairing_empty
+
+
+
+
+onepoint
+ecpairing_onepoint
+
+
+
+
+twopoint
+ecpairing_twopoint
+
+
+
+
 multipoint
-ecpairing
+ecpairing_multipoint
```

Comment:
Lets temporarily remove twopoint and multipoint cases from here as they take quite a bit of time.